src/main/java/pl/confitura/jelatyna/admin/Repository.java

```diff
@@ -0,0 +1,11 @@
+package pl.confitura.jelatyna.admin;
+
+import java.util.List;
+
+@org.springframework.stereotype.Repository("adminRepository")
+public interface Repository extends org.springframework.data.repository.Repository<Admin, Long> {
```

Comment:
1. widoczność pakietowa/domyślna - szczegół. Można zmienić

2. Nazwa - patrz wyżej

3. Rozszerzanie `Repository`: Jeśli interface rozszerza `JpaRepository` lub `CrudRepository` to implicite udostęnia wszystkie metody które są zadeklarowane w tych interfejsach - nawet jeśli ich nie potrzebuje. Dlatego też powinno się unikać tych interfaców. Zamiast tego użyć Repository i po prostu wypisać wszystkie te metody których się potrzebuje. W tym wypadku tylko  2 z pierdyliarda zupełnie nie potrzebnych.